admin-dev/themes/new-theme/js/pages/product/components/combination-list/combinations-list-renderer.ts

```diff
@@ -187,7 +187,7 @@ export default class CombinationsListRenderer {
     const $combinationsTableBody = $(CombinationsMap.combinationsTableBody);
 
     $combinationsTableBody.empty();
-    $combinationsTable.find(CombinationsMap.bulkSelectAll).prop('checked', false);
+    $combinationsTable.find(CombinationsMap.bulkSelectAllInPage).prop('checked', false);
```

Comment:
Maybe here you need to uncheck all the bulk checkboxes (including the ones in the dropdown)

Reply:
instead i moves this part to choices-selector component on "listRendered" event. to keep it in one place.